rar.go

```diff
@@ -94,7 +105,7 @@ func (r *Rar) Unarchive(source, destination string) error {
 			break
 		}
 		if err != nil {
-			if r.ContinueOnError {
+			if r.ContinueOnError || strings.Contains(err.Error(), "illegal file path") {
```

Comment:
I'd prefer that we expose `ErrIllegalPath` and check against that, however, I'm accepting this as-is and will save that for a later commit.